src/app/components/article/Articles.js

```diff
@@ -40,8 +39,12 @@ const ArticlesComponent = ({
   updateMutation,
   reloadData,
 }) => {
-  const [openEdit, setOpenEdit] = React.useState(false);
-  const [selectedArticle, setSelectedArticle] = React.useState(null);
+  let selectedArticleId = null;
+
+  if (type === 'fact-check') selectedArticleId = getQueryStringValue('factCheckId');
+  if (type === 'explainer') selectedArticleId = getQueryStringValue('explainerId');
+
+  const [selectedArticle, setSelectedArticle] = React.useState(selectedArticleId);
```

Comment:
Suggestion: I would suggest calling these `selectedArticleDbid` and `setSelectedArticleDbid`, respectively, for clarity. Otherwise, I would expect these to be an object. Similarly, `selectedArticleId` could be `selectedArticleDbidFromUrl`, something like that.

Reply:
Great suggestion. Applied it.